src/components/SwapWidgetSkeleton.tsx

```diff
@@ -0,0 +1,89 @@
+import { StrictMode } from 'react'
+import styled from 'styled-components/macro'
+import { Theme, ThemeProvider } from 'theme'
+
+import Column from './Column'
+import Row from './Row'
+import ReverseButton from './Swap/ReverseButton'
+import { WidgetWrapper } from './Widget'
+
+const LoadingWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  justify-content: space-between;
+`
+const Blob = styled.div<{ height: string; width: string; radius?: number; darker?: boolean }>`
+  background-color: ${({ darker, theme }) => (darker ? theme.outline : theme.module)};
+  border-radius: ${({ theme, radius }) => (radius ?? 0.25 * theme.borderRadius) + 'em'};
+  height: ${({ height }) => height};
+  width: ${({ width }) => width};
+`
+const WideColumn = styled(Column)`
+  width: 100%;
+`
+
+function FloatingDetails({ darker }: { darker?: boolean }) {
+  return (
+    <WideColumn gap={0.75}>
+      <Row>
+        <Blob height="16px" width="40px" darker={darker} />
```

Comment:
Widget expresses sizing in `em` - can you convert these all to `em` instead of `px` for consistency?